lms/security.py

```diff
@@ -116,9 +117,15 @@ def identity(self, request):
         userid = self.authenticated_userid(request)
 
         if userid:
-            return Identity(
-                userid, [Permissions.LTI_LAUNCH_ASSIGNMENT, Permissions.API]
-            )
+            permissions = [Permissions.LTI_LAUNCH_ASSIGNMENT, Permissions.API]
+
+            if any(
+                role in request.lti_user.roles.lower()
+                for role in ["administrator", "instructor", "teachingassistant"]
+            ):
+                permissions.append(Permissions.LTI_CONFIGURE_ASSIGNMENT)
```

Comment:
Nice, we under utilize the security part of security.py/security policies and abuse it for other things.